### DISCUSSION TITLE
Mic efficiency

### DISCUSSION THREAD

**Discussion by bilal201715** [speechly/speechly#322] [Q&A]
Hi sir, I built an e-commerce store with Speechly but the voice recognition is not as efficient as your e-commerce demo.

I think the problem is that I'm using the free version, right? Please guide me, thanks

My site link:

https://nova-store.vercel.app/

**Comment by thisismatu** [comment]
Hi @bilal201715 👋 

I don't think the issue is the free version but something else. Perhaps this is related to your implementation or the training data. The ecommerce demo source code is available here https://github.com/speechly/speechly/tree/main/demos/fashion-ecommerce Maybe take a look at our implementation to get some ideas on how you might improve your application.
